reesegreen2014/rancid-tomatillos
# Testing/cypress

#### What does this PR do?
This pull request implements Cypress testing for happy and sad path user flows.

#### How should this be manually tested?
`npx cypress open`
select browser 
select E2E testing
navigate to Cypress command log to see all tests are passing

#### Any background context you want to provide?
No
#### What are the relevant tickets (if any) ?
None
#### Screenshots (if appropriate)
N/A
#### Questions:
No questions at this time.
